assets/js/modules/adsense/components/dashboard/AdSensePerformanceWidget.js

```diff
@@ -103,7 +103,7 @@ class AdSensePerformanceWidget extends Component {
 				className: 'googlesitekit-data-block--impression',
 				title: __( 'Page CTR', 'google-site-kit' ),
 				/* translators: %s: percentage value. */
-				datapoint: sprintf( _x( ' %1$s%%', 'AdSense performance Page CTA percentage', 'google-site-kit' ), numberFormat( currentRangeData.totals[ 3 ] * 100, { maximumFractionDigits: 2 } ) ),
+				datapoint: sprintf( _x( ' %1$s', 'AdSense performance Page CTA percentage', 'google-site-kit' ), numberFormatWithUnit( currentRangeData.totals[ 3 ] * 100, '%', { maximumFractionDigits: 2 } ) ),
```

Comment:
There's no need for this translation string anymore now, let's remove it. Based on the other comments and changes, this can simply become
```suggestion
				datapoint: currentRangeData.totals[ 3 ] * 100,
```
The percentage should be passed in the `datapointUnit` prop.